libcontainer/user/user.go

```diff
@@ -11,16 +11,17 @@ import (
 )
 
 const (
-	minId = 0
-	maxId = 1<<31 - 1 //for 32-bit systems compatibility
+	minID = 0
+	maxID = 1<<31 - 1 // for 32-bit systems compatibility
```

Comment:
I think I have a similar change in https://github.com/opencontainers/runc/pull/2975